focuspoints.lrdevplugin/ExifUtils.lua

```diff
@@ -30,8 +31,7 @@ exiftoolWindows = LrPathUtils.child(exiftoolWindows, "exiftool.exe")
 
 function ExifUtils.getExifCmd(targetPhoto)
   local path = targetPhoto:getRawMetadata("path")
-  local metaDataFile = LrPathUtils.removeExtension(path)
-  metaDataFile = metaDataFile .. "-metadata.txt"
+  local metaDataFile = LrPathUtils.child(LrPathUtils.getStandardFilePath("temp"), LrUUID.generateUUID() .. ".txt")
```

Comment:
What we could do btw is to add a DEBUG mode.
It the case here, instead of using UUID and deleting the file we could then use the photo file name and don't delete the file when in Debug.